firstlastelement.py

```diff
@@ -0,0 +1,42 @@
+class Solution:
+    def searchRange(self, nums: List[int], target: int) -> List[int]:
+        fIndex=  self.firstIndex(target,nums)
+        Lindex=  self.LastIndex(target,nums)
+        return(fIndex,Lindex)
+   
+   
+        # we are going to do a bunary search, compare the middle element with the target and move the pointer accordingly and then conduct binary search in that range
+    def firstIndex(self, target, nums):
+        low=0
+        high=len(nums)-1
+        while(low<=high):
+            mid= (low+high)//2  #finding the middle index
```

Comment:
Just make it habit to use low+(high-low)/2 instead.
It doesnot give error in python but you should know this.